tcms/testplans/static/testplans/js/get.js

```diff
@@ -70,6 +62,25 @@ function getTestCaseRowContent(rowContent, testCase, permissions) {
         row.find('.js-test-case-menu-clone')[0].href = `/cases/clone/?case=${testCase.id}`;
     }
 
+    var status_indication_element = row.find('.js-test-case-status');
+
+    // apply visual separation between confirmed and not confirmed
+    //todo: refactor when testcase_status is replaced with boolean flag
+    if (testCase.case_status_id !== 2) {
+        row.find('.list-group-item-header').addClass('bg-warning');
+        status_class_to_remove = 'fa-check';
+        status_as_text = 'Not Confirmed';
+    }else {
+        row.find('.list-group-item-header').addClass('bg-success');
+        status_class_to_remove = 'pficon-close';
+        status_as_text = 'Confirmed';
```

Comment:
I'm not decided for the icons but the text needs to be translated and b/c ATM there are 4 statuses you need to take into account all of them. IMO make it simple and drop the icon and hint, just leave the red color for unconfirmed cases. Then we'll see how users respond to that.